js/tweets.js

```diff
@@ -3,80 +3,80 @@
-		document.getElementsByClassName('tweets-feed')[0].style.opacity =  0;
-		window.setTimeout(parseFunc, 560);
-	}
+    if (tweetNum > 0) {
+        tweetNum -= 1;
+        Interval();
+        document.getElementsByClassName('tweets-feed')[0].style.opacity =  0;
+        window.setTimeout(parseFunc, 560);
+    }
 }
 
 function parser(data) {
-	var parsed = ""
-	var tweet = data.statuses[tweetNum].text;
-	var words = tweet.split(" ");
-	var loklakLinkCount = 0;
-	for (word in words) {
-		if (words[word].startsWith("@")) {
-			parsed += "<a href='https://twitter.com/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
-		} else if (words[word].startsWith("#")) {
-			parsed += "<a href='https://twitter.com/hashtag/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
-		} else if (words[word].startsWith("http")) {
-			if (words[word].startsWith("http://loklak")) {
-				parsed += "<a href='" + data.statuses[tweetNum].links[loklakLinkCount] + "' target='_blank'>" + data.statuses[tweetNum].links[loklakLinkCount] + "</a> ";
-				loklakLinkCount += 1;
-			} else {
-				parsed += "<a href='" + words[word] + "' target='_blank' style='word-break:break-all'>" + words[word] + "</a> ";
-			}
-		} else {
-			parsed += words[word] + " ";
-		}
-	}
-  var date = +new Date(data.statuses[tweetNum].created_at);
-  var myDate = new Date();
-  myDate = myDate.getTime();
-  var difference = myDate-date; //it's in miliseconds
-  var tweetAge = Math.round(difference/1000/3600) > 24 ? Math.round(difference/1000/3600/24) + " days ago" : Math.round(difference/1000/3600) + " hours ago";
-  parsed += "<span class='dateTweeted'>Tweeted "+tweetAge+"</div>";
-
-	document.getElementsByClassName("tweets-feed")[0].innerHTML =  parsed;
-	document.getElementsByClassName("tweets-feed")[0].style.opacity =  1;
+    var parsed = ""
+    var tweet = data.statuses[tweetNum].text;
+    var words = tweet.split(" ");
+    var loklakLinkCount = 0;
+    for (word in words) {
+        if (words[word].startsWith("@")) {
+            parsed += "<a href='https://twitter.com/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
+        } else if (words[word].startsWith("#")) {
+            parsed += "<a href='https://twitter.com/hashtag/" + words[word].slice(1) + "' target='_blank'>" + words[word] + "</a> ";
+        } else if (words[word].startsWith("http")) {
+            if (words[word].startsWith("https://loklak")) {
+                parsed += "<a href='" + data.statuses[tweetNum].links[loklakLinkCount] + "' target='_blank'>" + data.statuses[tweetNum].links[loklakLinkCount] + "</a> ";
+                loklakLinkCount += 1;
+            } else {
+                parsed += "<a href='" + words[word] + "' target='_blank' style='word-break:break-all'>" + words[word] + "</a> ";
+            }
+        } else {
+            parsed += words[word] + " ";
+        }
+    }
+    var date = +new Date(data.statuses[tweetNum].created_at);
```

Comment:
Is the `+` sign needed before `new Date` ?

Reply:
@akshatnitd , I have copied this entire code from https://github.com/fossasia/2018.fossasia.org/blob/master/js/tweetsfetcher.js

In the https://github.com/fossasia/2018.fossasia.org repo there exist two js files one named 
tweet.js and another tweetfetcher.js . 

Basically the previously existing tweet.js is almost the same as the tweet.js of the above mentioned repo of 2018.fossasia.org.

They only differ by this

for tweet.js of 2018.fossasia.org
 `function parseFunc(){
    parser(stuff)
}`
and
for tweet.js of fossasia.org
`function parseFunc(stuff){
    parser(stuff)
}`

In the `nextTweet()` function when `parseFunc` is called it is called without any paramter as
` window.setTimeout(parseFunc, 560)`

Also when I logged the url in the console
for both sites i.e **fossasia.org** and **2018.fossasia.org** I found out that the search query of the fossasia.org fetched only one data item whereas 2018.fossasia.org fetches three data items .
That is why when you open the console of the browser in the website fossasia.org you can see a number of errors which occur due to the fetching of one data item. Thus this leads to data becoming undefined and starts causing errors. 

That is why as you can see in the index.html file 

that I have changed `data-from` to `data-query`

because the search query generated with `data-from` when entered in the loklak API 
fetches one data-item

I tried fixing this but could work my head on it.

And as @mariobehling has suggested he wants the site of fossasia.org to function the same way as 2018.fossasia.org , so that is why I made the changes

Reply:
Okay. Leave this change and make rest of suggested changes